### PR TITLE
feat(setup): update logo and components props

### PR DESCRIPTION
**Pull request by mikeplascdev**
#### Pull request type

- [x] Bug Fix
- [x] Enhancement
- [ ] Documentation

### Description

- Update logos
- Add new props to the Accordion and Button components

### Figma
[Console-Design](https://www.figma.com/design/a3qQCRxNWxP7K1titmso0e/Console-Design?node-id=0-1&t=kNnJ9JHwCzbr7Ik7-0)

### Evidence

https://github.com/theinfinitereality/eepro-multitenancy/assets/113119956/22858014-8f00-4a91-9f4e-62c1375d8f9b

### Jira
[IR-2119](https://tsu.atlassian.net/browse/IR-2119)

[IR-2119]: https://tsu.atlassian.net/browse/IR-2119?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/client/scripts/getClientSettings.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const getClientSetting = async () => {
     .then(([dbClient]) => {
       const dbClientConfig = clientDbToSchema(dbClient) || {
         logo: './logo.svg',
-        title: 'Ethereal Engine',
+        title: 'IR Engine Console',
```
Comment (HexaField) [inline packages/client/scripts/getClientSettings.ts]: Should read just `IR Engine`

**Review by HexaField — CHANGES_REQUESTED** [review]

**Review by hanzlamateen — APPROVED** [review]

File: packages/client/scripts/getClientSettings.ts (hunk 1 above)
Comment (mikeplascdev) [reply packages/client/scripts/getClientSettings.ts]: Done

**Comment by HexaField** [comment]
I'd like @aditya-mitra 's eyes on this before we merge in, as there are some changes to the tailwind primitives.

File: packages/ui/src/primitives/tailwind/Button/index.tsx [hunk 2]
```diff
@@ -24,14 +24,15 @@ Ethereal Engine. All Rights Reserved.
...
+
 import { twMerge } from 'tailwind-merge'
 
 export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   startIcon?: ReactNode
   endIcon?: ReactNode
   children?: ReactNode
   size?: 'small' | 'medium' | 'large'
-  variant?: 'primary' | 'outline' | 'danger' | 'success' | 'pink' | 'transparent' | 'onboarding'
+  variant?: 'primary' | 'outline' | 'danger' | 'success' | 'pink' | 'transparent' | 'onboarding' | 'sidebar'
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Button/index.tsx:35]: Is a `sidebar` variant present in the primitive figma designs? 
We should only add colour variant which are primitives. (I plan to refactor the `onboarding` variant)

For any other variant, we should add it as a separate derived button. Something like this
```tsx
const SidebarButton = (props: ButtonProps) => <Button className='bg-[#141619] {...props} />
```

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Looks good!
A small modification in the tailwind button would be nice.

**Review by hanzlamateen — APPROVED** [review]

File: packages/ui/src/primitives/tailwind/Button/index.tsx (hunk 2 above)
Comment (hanzlamateen) [reply packages/ui/src/primitives/tailwind/Button/index.tsx:35]: I am merging this PR for now as I want to deploy this in MT. @mikeplascdev can you please address aditya's concern in a separate PR.

**Comment by DanielBelmes** [comment]
Since the urls have stayed the same. We won't be able to see the updated assets on domains we visited before. In my PR I added query parameters to the urls to cache bust them. Whoever is giving demos Wednesday needs to clear their cache to be able to see these changes. @hanzlamateen

**Pull request merged**